### ISSUE TITLE
Logisim is which one?

### ISSUE THREAD

**Issue by quantrpeter** [logisim-evolution/logisim-evolution#1703]
Hi
   Logisim is which one? I believe it is event-driven simulator, right?
thanks

<img width="510" alt="image" src="https://user-images.githubusercontent.com/50471577/229339706-97c6838c-b1ce-46f7-bbd2-7981c41e0509.png">

**Comment by maehne** [comment]
Yes, Logisim-evolution is an event-driven simulator. You can learn more of its inner working by reading the chapter "Value propagation" in the "Guide to Being a Logisim User" (Help -> User's Guide).
